packages/core/src/interfaces/charts.ts

```diff
@@ -34,14 +34,18 @@ export interface BaseChartOptions {
 	 * legend configuration
 	 */
 	legend?: LegendOptions;
+	/**
+	 * Optional function to determine whether is filled based on datasetLabel, label, and/or value
+	 */
+	getIsFilled?: (datasetLabel: any, label?: any, value?: any, data?: any, defaultFilled?: boolean) => boolean;
 	/**
 	 * Optional function to generate the fill color based on datasetLabel, label, and/or value
 	 */
-	getFillColor?: (datasetLabel: any, label?: any, value?: any) => string;
+	getFillColor?: (datasetLabel: any, label?: any, value?: any, data?: any, defaultFillColor?: string) => string;
 	/**
 	 * Optional function to generate the stroke color based on datasetLabel, label, and/or value
 	 */
-	getStrokeColor?: (datasetLabel: any, label?: any, value?: any) => string;
+	getStrokeColor?: (datasetLabel: any, label?: any, value?: any, data?: any, defaultStrokeColor?: string) => string;
```

Comment:
we discussed changing all of these to just receive a `d`, and maybe a `defaultValue`

Reply:
good catch, I had to leave the datasetLabel, and label separate from the data parameter because for tooltips they use the same interface (without having the full datapoint), but I have combined value and data into the same parameter